modules/core/src/matmul.simd.hpp

```diff
@@ -1606,7 +1606,7 @@ transform_16u( const ushort* src, ushort* dst, const float* m, int len, int scn,
 static void
 transform_32f( const float* src, float* dst, const float* m, int len, int scn, int dcn )
 {
-#if CV_SIMD && !defined(__aarch64__) && !defined(_M_ARM64)
+#if CV_SIMD
```

Comment:
> transform_32f

Should be `transform_16u` as stated here: https://github.com/opencv/opencv/issues/19163#issuecomment-748471830